.gitignore

```diff
@@ -15,3 +15,10 @@ project/plugins/project/
 # Scala-IDE specific
 .scala_dependencies
 .worksheet
+.idea/
+.idea_modules/
+*.iml
+
+# Platform specific
+.DS_Store/
```

Comment:
i have used it as a matter of habit to avoid accidently committing `.DS_STORE` files  in any git repo. Is there a better way to avoid it?

Maybe a global git config exclude filter but I thought this is safer approach, what you think?